workspace_tools/toolchains/iar.py

```diff
@@ -66,10 +66,10 @@ def __init__(self, target, options=None, notify=None, macros=None, silent=False,
             self.asm  = [join(IAR_BIN, "iasmarm")] + ["--cpu", cpuchoice]
         if not "analyze" in self.options:
             self.cc   = [main_cc] + c_flags
-            self.cppc = [main_cc, "--c++",  "--no_rtti", "--no_exceptions"] + c_flags
+            self.cppc = [main_cc, "--c++",  "--no_rtti", "--no_exceptions", "--guard_calls"] + c_flags
```

Comment:
Note to myself: we need to clean up IAR flags. otherwise when you export you wont get all these new flags in the project file :sob: